app/controllers/api/authorizations_controller.rb

```diff
@@ -8,4 +8,8 @@ class Api::AuthorizationsController < Api::BaseController
   def resource
     authorization
   end
+
+  def show_cache_path
+    authorization.cache_key
```

Comment:
I removed all the `count`s from the `/api/v1/authorization` JSON.  So now the only thing that varies is the user_id - so it's safe to cache based on this key.